swagger.yaml

```diff
@@ -6840,6 +6840,10 @@ definitions:
         type: string
       user_name:
         type: string
+      labels:
+        x-go-custom-tag: gorm:"type:text"
+        type: string
+        description: Json containing host's labels.
```

Comment:
Is this a generic labeling system, or only for enabling ODF?

Reply:
As far as I remember from our conversation there was no need for generic node labeling. It is ODF sepcific.

Reply:
So this should be odf_enabled (boolean)